pkg/karmadactl/cmdinit/config/types.go

```diff
@@ -0,0 +1,354 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "config.karmada.io"
+
+// SchemeGroupVersion is group version used to register these objects
+var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+
+// KarmadaInitConfig defines the configuration for initializing Karmada
+type KarmadaInitConfig struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
```

Comment:
I think this is not need, let's delete it.

Reply:
Do you mean the `metav1.ObjectMeta`? It makes sense to me.
Please add a task to the tracking issue.

Reply:
Updated,  How about including it in https://github.com/karmada-io/karmada/pull/5306#issuecomment-2441115638 (for Summer OSPP 2024)?,  or alone PR for it

Reply:
Both are OK to me. I just want somewhere track this.